docs/components/tasklist/userguide/using-tasklist.md

```diff
@@ -1,113 +1,89 @@
 ---
 id: using-tasklist
-title: Overview and example use case
-description: "What you can do with Tasklist and an example use case."
+title: Overview
+description: "What you can do with Tasklist."
 ---
 
-## What can I do with Tasklist?
+import TasklistTasksPageSpecifications from '../img/tasklist-page-specifications.png';
+import TaskTileSpecification from '../img/task-tile-specification.png';
+import styles from "./styles.module.css";
 
-Tasklist shows you all user tasks that appeared in processes. Those processes are running in Zeebe.
+Tasklist provides a user-friendly interface for managing and completing tasks that require manual interaction.
+It shows you all user tasks that appear in processes. Those processes are running in [Zeebe](/docs/components/zeebe/zeebe-overview.md).
 
-User tasks need an interaction from the user. This can be updating, adding variables, filling out a [Camunda Form](../../../guides/utilizing-forms.md), or simply completion of the task.
+The user interaction with a task may involve making updates, adding variables, filling out a [Camunda Form](../../../guides/utilizing-forms.md), or simply reviewing and completing the task.
 
-To work on a task the user must self-assign it first. Once assigned, the task can be completed.
-The user can unassign the task if they do not intend to work on it.
+User tasks can be automatically assigned to users and groups in the BPMN process or they must be self-assigned from Tasklist.
+Once assigned to a user, the task can be completed. The user can unassign the task if they do not intend to work on it.
 
-Different task statuses and filters help the user choose the desired task.
-
-:::note
+:::info
 When a user is granted Tasklist access, the user has full access to the respective process instance data.
 :::
 
-## Example use case
-
-When you've successfully logged in, you'll see a screen similar to the following:
+## Tasks overview
 
-![tasklist-start-screen](../img/tasklist-start-screen_light.png)
+Tasklist has two main pages:
 
-On the left side of the screen, you can see the list of tasks. On the right side of the screen, you can see details of the currently selected task.
+- Tasks page (_described here_).
```

Comment:
Are we linking to something here?

Reply:
No. I removed the note "_described here_"